DuckDuckGo/PrivacyProtectionController.swift

```diff
@@ -66,6 +65,7 @@ class PrivacyProtectionController: ThemableNavigationController {
             PrivacyProtectionErrorController(coder: coder, configuration: self.privacyConfig)
         }) else { return }
         controller.errorText = errorText
+        controller.delegate = self
```

Comment:
Seems like the delegate setting when a page displays error was missing here, was it intentional?